### PR TITLE
Apple hard refresh is actually cmd+option+R

### PR DESCRIPTION
**Pull request by gStart9**

### CODE REVIEW

File: site/source/user-manual/getting-started/initial-setup.rst [hunk 1]
```diff
@@ -87,7 +87,7 @@ If you are experiencing issues with setup, try the following:
     - Mullvad - Go to "Settings -> VPN Settings -> Local Network Sharing"
     - ProtonVPN - Go to "Preferences -> Connection -> Allow LAN Connections"
 
-#. Visit or refresh (ctrl+shift+R - Linux/Windows, cmd+shift+R - Mac) the start.local page in a web browser
+#. Visit or refresh (ctrl+shift+R - Linux/Windows, cmd+option+R - Mac) the start.local page in a web browser
```
Comment (gStart9) [inline site/source/user-manual/getting-started/initial-setup.rst:90]: A user has told us cmd+shift+r on a mac is "read mode" and cmd+option+R is the hard refresh

Comment (13Homer13) [reply site/source/user-manual/getting-started/initial-setup.rst:90]: This work for safari only. FF still uses cmd+shift+R.

Comment (gStart9) [inline site/source/user-manual/getting-started/initial-setup.rst:90]:
```suggestion
#. Visit or refresh (ctrl+shift+R on Linux/Windows, cmd+shift+R on macOS in Firefox, or cmd+option+R on macOS in Safari) the start.local page in a web browser
```

Comment (gStart9) [reply site/source/user-manual/getting-started/initial-setup.rst]: TY homer.  Updated.

**Comment by kn0wmad** [comment]
This will need to be rebased unfortunately

**Comment by gStart9** [comment]
Closing in favor of #477

**Pull request closed without merging**